include/eve/detail/function/simd/x86/friends.hpp

```diff
@@ -136,7 +136,7 @@ namespace eve::detail
       else
       {
         constexpr auto use_avx2 = current_api >= avx2;
-        constexpr auto eq = []<typename E>(E e, E f) { return as_logical_t<E>(e == f); };
+        constexpr auto eq = []<typename E>(E const& e, E const& f) { return as_logical_t<E>(e == f); };
```

Comment:
I an worrying about lambdas.
Is this for aggregate case only now?

It's not a new problem we can fix it when we get there.

Reply:
I'll probably do a pass on all those comparisons, make it manual for 64 bits integral and remove the lambda.
Next PR ?